modules/nodes-scheduler-default-creating.adoc

```diff
@@ -140,4 +137,3 @@ $ oc logs openshift-kube-scheduler-ip-10-0-141-29.ec2.internal | grep predicates
 
 Creating scheduler with fit predicates 'map[MaxGCEPDVolumeCount:{} MaxAzureDiskVolumeCount:{} CheckNodeUnschedulable:{} NoDiskConflict:{} NoVolumeZoneConflict:{} MatchNodeSelector:{} GeneralPredicates:{} MaxCSIVolumeCountPred:{} CheckVolumeBinding:{} MaxEBSVolumeCount:{} PodFitsResources:{} MatchInterPodAffinity:{} HostName:{} PodToleratesNodeTaints:{}]' and priority functions 'map[InterPodAffinityPriority:{} LeastRequestedPriority:{} ServiceSpreadingPriority:{} ImageLocalityPriority:{} SelectorSpreadPriority:{} EqualPriority:{} BalancedResourceAllocation:{} NodePreferAvoidPodsPriority:{} NodeAffinityPriority:{} TaintTolerationPriority:{}]'
```

Comment:
@bergerhoffer one question here, when the above four predicates come under general predicates and not possible to declare it multiple times, would the log have reference to both general predicates and the four predicates which come under it  like shown aboave MatchNodeSelector , HostName  etc ? or would it have either of them ? Can you please confirm once ?

Reply:
@soltysh Can you confirm, should `PodFitsResources:{}`, `MatchNodeSelector:{}`, and `HostName:{}` be removed from this example output? Or would all 4 also be listed here?

Reply:
@kasturinarra is right, they should be removed as well.

Reply:
Also while you're at it a few lines above here there's a typo:
`checks for the predoicates and priorites` 
should be 
`checks for the predicates and priorites`